gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -27,31 +26,34 @@ use tokio_stream::wrappers::ReceiverStream;
 use tonic::{transport::Server, Status};
 use tracing::error;
 
+#[derive(Parser)]
+pub struct ClnExtensionOpts {
+    /// Gateway CLN extension service bind address
+    #[arg(long = "addr", env = "GW_CLN_EXTENSION_BIND_ADDRESS")]
+    pub addr: SocketAddr,
+}
```

Comment:
Optionally? Sure. But this will be horribly annoying to set up with e.g. nix-bitcoin where I can't simply inject env vars into the cln service afaik. Can we just make it a cln config option and maybe allow overriding via env? thoughts @dpc? I agree with the removal of the config file.

Reply:
We'd have to define a default address in case a user doesn't care to supply cln config. Also, it might be a painful experience to always require a the cln plugin config even when env var is set

Reply:
We wouldn't require it, just make it optional by giving it a clearly invalid default. If neither is specified we can either fall back to a default or stop with an error message. We'll need some more config anyway to authenticate requests imo.